src/exo/LoopIR_compiler.py

```diff
@@ -434,17 +434,17 @@ def from_lines(x):
 // Compiler feature macros adapted from Hedley (public domain)
 // https://github.com/nemequ/hedley
 
-#if defined(__has_extern)
-#  define EXO_HAS_extern(extern) __has_extern(extern)
+#if defined(__has_builtin)
```

Comment:
All these macros should be reverted back to what they used to be. The reason they exist is orthogonal to your PR.

Reply:
Yeah was doing that